### PR TITLE
Implemented send_tcp and handle functions

### PR DESCRIPTION
**Pull request by athata1**
I implemented the send_tcp function as well as handle function. 

To the best of my knowledge, both seem to work fine, but a second look wouldn't hurt.

### CODE REVIEW

File: src/shallot.py [hunk 1]
```diff
@@ -26,6 +29,9 @@ def send_tcp(ip: str, port: int, data: bytes):
     if ip == list_server.my_public_ip:
         ip = 'localhost'
     # TODO open a TCP socket to the (IP, port), send the data, and close the connection
+    with socketserver.socket(socketserver.AF_INET, socketserver.SOCK_STREAM) as sock:
```
Comment (AnugunjNaman) [inline src/shallot.py]: I don't think socketserver has AF_INET or SOCKSTREAM ? It's a socket library that has it. Did you test it? If so I would like to know how.

Even if I modify your code to correct purpose, it really hard to test the whole code base in general. The `make_request` function is not working well. Essentially after `send_tcp`, the handle() is not being called from `ShallotHandler` class.

File: src/shallot.py [hunk 2]
```diff
@@ -9,6 +9,9 @@
 import crypto
 import list_server
 from config import SHALLOT
+from config import X25519_SIZE
```
Comment (AnugunjNaman) [inline src/shallot.py]: You probably want to call `CRYPTO` from config here. This doesn't import well.

File: src/shallot.py (hunk 1 above)
Comment (athata1) [reply src/shallot.py]: Yeah this was an autocomplete typo. I fixed the issue and pushed code.

File: src/shallot.py (hunk 2 above)
Comment (athata1) [reply src/shallot.py]: Resolved this

**Comment by AnugunjNaman** [comment]
Sort #10 #9 #8 #7 .

**Pull request merged**